.eslintrc.js

```diff
@@ -35,6 +35,7 @@ module.exports = {
       files: ['*.ts', '*.tsx'],
       rules: {
         'react/prop-types': 'off',
+        'no-console': ['warn'],
```

Comment:
maybe change this to error? then husky shouldn't let us push console.logs? we need to make exceptions for files that must contain a logger, e.g. the `use-logger` hook.

Reply:
Yes! I like it being an error